### ISSUE TITLE
New versions of numpy don't have an expand_dims method on ndarrays

### ISSUE THREAD

**Issue by wmeddie** [SkymindIO/skil-python#30]
Looks like in 1.15 you have to use np.expands_dims(arraylike, index) instead.  If this function is available in older versions of numpy we should use it instead

**Comment by maxpumperla** [comment]
for context: 

Looking at the example, you can use the keras model and the tf model, the former can save directly to the file, and the latter uses the freeze_graph in the tensorflow tools to save the model. The input is one or more arrays, which should eventually be converted to INArray and then serialized to the interface. Here encountered a problem, from the process of np.ndarray conversion, the example directly calls: data.expand_dims, after consulting the document, it may be np.expand_dims (data, 0). If the dims of array are incorrect, an error is returned, but there are no specific hints, such as the received array dimension and the actual accepted dimensions. I am currently stuck here and I want to confirm it.

**Comment by maxpumperla** [comment]
@wmeddie to me that looks like the user wants better error handling when the call fails, in particular for the second argument (axes=0 above). scanning release notes, `expand_dims` is still there, but has a deprecation warning on unsupported axis ranges.

@farizrahman4u can you please have a look at this?
